### ISSUE TITLE
bug in ApiAiAssistant.getUser()

### ISSUE THREAD

**Issue by germasch** [actions-on-google/actions-on-google-nodejs#8]
The logic in ApiAiAssistant to check whether the user object exists in the incoming request is missing parentheses, breaking it in the case that body_.originalRequest exists, but not data or data.user.

The patch below fixes it. I can make a pull request / whatever, but it seems easier if someone were to just fix this directly.

```
diff --git a/actions-on-google.js b/actions-on-google.js
index 01b9c30..e0d2f98 100644
--- a/actions-on-google.js
+++ b/actions-on-google.js
@@ -1492,9 +1492,9 @@ ApiAiAssistant.prototype = new Assistant();
 ApiAiAssistant.prototype.getUser = function () {
   debug('getUser');
   let self = this;
-  if (!self.body_.originalRequest &&
-      self.body_.originalRequest.data &&
-      self.body_.originalRequest.data.user) {
+    if (!(self.body_.originalRequest &&
+         self.body_.originalRequest.data &&
+         self.body_.originalRequest.data.user)) {
     self.handleError_('No user object');
     return null;
   }
```

**Comment by smishra2** [comment]
Thanks for catching this, @germasch. Will be fixed in the next release.

**Comment by entertailion** [comment]
Fixed in 1.0.5
